core/src/main/scala/cats/syntax/applicative.scala

```diff
@@ -17,3 +19,8 @@ final class ApplicativeOps[F[_], A](private val fa: F[A]) extends AnyVal {
   def unlessA(cond: Boolean)(implicit F: Applicative[F]): F[Unit] = F.unlessA(cond)(fa)
```

Comment:
I also suggest deprecating these two

Reply:
You could just make the old syntax not implicit and substitute the new one with the same names.

Reply:
@joroKr21 then `replicateA` will become a syntax extenstion on` => F[A]`, while the method itself accepts `F[A]`. Is it ok?

Reply:
Hmm I'm not sure

Reply:
@catostrophe how about `replicateA(0)`? Shouldn't that also **not** execute `fa`?

Reply:
Yes, but replicateA in Applicative takes a call-by-value param

Reply:
Yes, and exposing a by-name API for `replicateA` might mislead users to think of a more imperative style of API like `List.fill` where `=> F[A]` might have side-effects.

Another option would be to extract `ApplicativeStrictOps` for `replicateA`